### ISSUE TITLE
"longer description" and "breaking changes" are not included in the commit message

### ISSUE THREAD

**Issue by Malkiz** [commitizen/cz-conventional-changelog#13]
I am using Windows 7, and when running the `git cz` command with the `cz-conventional-changelog` adapter - the "longer description" and "breaking changes" are not included in the commit message.

I debugged it a little and found that the problem is with concatenating the `\n\n` to the message.

``` javascript
commit(head + '\n\n' + body + '\n\n' + footer);
```

Anything after a `\n` character is not included in the commit.
It works if I switch it to commas like this:

``` javascript
commit(head + ',' + body + ',' + footer);
```

https://github.com/commitizen/cz-conventional-changelog/blob/master/index.js#L98

**Comment by jimthedev** [comment]
Interesting. I'll take a look to see if I can reproduce. I've been working on core a lot and this adapter certainly needs some updates and likely should have a test suite. In the mean time I'll try to get a fix out for 1.1.5.

**Comment by Filipoliko** [comment]
Hi, just ran into this issue with Windows 10, `git-cz` v4.7.1 and `cz-conventional-changelog` v3.3.0. Has this not been fixed yet?
